src/viewport.cpp

```diff
@@ -3229,6 +3252,7 @@ calc_heightdiff_single_direction:;
 			FALLTHROUGH;
 
 		case VPM_X_AND_Y: // drag an X by Y area
+measure_area:
```

Comment:
Should this have a semicolon after it? (The example above does. I've never seen a `goto` in the wild before so I don't know. 😄 )

Reply:
This is correct, labels are suffixed with colons, not semicolons.